scripts/check_compliance.py

```diff
@@ -1002,12 +1005,14 @@ def set_commit_status(status, msg):
             set_commit_status(
                 'error', 'Error during verification, please report!')
 
-        # Always show informational messages. See ComplianceTest.add_info() for
-        # an explanation of the [newline] thing.
+        # Always show informational messages. See ComplianceTest.add_info() re.
+        # the decoding thing. We undo it here.
         if case.info_msg:
             comment += "## {} (informational only, not a failure)\n\n" \
                        "```\n{}\n```\n\n".format(
-                           case.name, case.info_msg.replace("[newline]", "\n"))
+                           case.name,
+                           case.info_msg.encode("utf-8")
+                                        .decode("unicode-escape"))
```

Comment:
In the future a `get_info()` accessor would be more consistent.

Reply:
Yeah, will def. add a helper for it if it needs to be unescaped in more places.